examples/src/main/java/com/mapbox/navigation/examples/core/MapboxBuildingHighlightActivity.kt

```diff
@@ -176,88 +199,14 @@ class MapboxBuildingHighlightActivity : AppCompatActivity(), OnMapLongClickListe
         mapboxNavigation.startTripSession()
         mapboxReplayer.pushRealLocation(this, 0.0)
         mapboxReplayer.play()
-
-        /**
-         * Try attaching building highlights to the arrival experience.
-         * At any point after constructing MapboxNavigation, enable the building arrival api
-         * It is disabled by default, and should be disabled at some point after enabling.
-         */
-        buildingsArrivalApi.enable(mapboxNavigation)
     }
 
     @SuppressLint("MissingPermission")
     private fun initStyle() {
         mapboxMap.loadStyleUri(
             MAPBOX_STREETS
-        ) { style ->
+        ) {
             binding.mapView.gestures.addOnMapLongClickListener(this)
-
-            /**
-             * Showing all buildings is disabled by default. But this function
-             * allows you to show all buildings in 3D with the highlighted one.
-             */
-            binding.toggleBuildings.setOnCheckedChangeListener { _, isChecked ->
-                extrudeBuildings(isChecked)
-            }
-
-            /**
-             * Try attaching to the map click listener, highlight the building the
-             * user has selected. This observer will not influence the callback from
-             * [MapboxBuildingArrivalApi.registerBuildingHighlightObserver].
-             */
-            binding.mapView.gestures.addOnMapClickListener { point ->
-                buildingHighlightApi.highlightBuilding(point, buildingHighlightObserver)
-                false
-            }
-        }
-    }
-
-    private val buildingHighlightObserver = BuildingHighlightObserver { features ->
-        updateFillBuildingsLayer(features)
-    }
-
-    /**
-     * Example map layer to highlight all buildings.
-     */
-    private fun updateFillBuildingsLayer(features: List<QueriedFeature>) {
```

Comment:
Can we keep this around? Extruding all buildings is all over the javascript examples but we don't have one in the android examples.

Also think it's best to show this working with the highlighted building example

Reply:
I wouldn't worry about it here since this examples module is gonna go away. We can add a separate building extrusions example to our new examples repository but isn't that more of a Maps example to extrude all buildings irrespective of highlighting just a one. 🤔

/cc @kmadsen @lukaszpaczos